helm_deploy/hmpps-interventions-service/templates/cronjob-data-extractor-analytics.yaml

```diff
@@ -13,7 +13,7 @@ spec:
           - name: data-extractor-analytics
             image: ministryofjustice/data-engineering-data-extractor:v1
             imagePullPolicy: Always
-            args: ["extract_table_names.py && extract_pg_jsonl_snapshot.py && transfer_local_to_s3.sh"]
+            args: ["extract_psql_all_tables_to_jsonl.sh && extract_pg_jsonl_snapshot.py && transfer_local_to_s3.sh"]
```

Comment:
@PriyaBasker23 should `extract_pg_jsonl_snapshot.py` be removed as well?

Reply:
Thanks for spotting. removed it now. We dont need it